pkg/controllers/cloudentityselector/controller_test.go

```diff
@@ -112,24 +114,80 @@ var _ = Describe("CloudEntitySelector Controller", func() {
 			mockAccManager.EXPECT().RemoveResourceFiltersFromAccount(&testAccountNamespacedName,
 				&testSelectorNamespacedName).Return().Times(1)
 
-			err := cesReconciler.processCreateOrUpdate(selector, &testSelectorNamespacedName)
+			err := reconciler.processCreateOrUpdate(selector, &testSelectorNamespacedName)
 			Expect(err).ShouldNot(HaveOccurred())
-			err = cesReconciler.processDelete(&testSelectorNamespacedName)
+			err = reconciler.processDelete(&testSelectorNamespacedName)
 			Expect(err).ShouldNot(HaveOccurred())
 		})
-
 		It("CES Add failure", func() {
 			mockAccManager.EXPECT().AddResourceFiltersToAccount(&testAccountNamespacedName, &testSelectorNamespacedName,
 				selector).Return(false, fmt.Errorf("dummy")).Times(1)
 			mockAccManager.EXPECT().RemoveResourceFiltersFromAccount(&testAccountNamespacedName,
 				&testSelectorNamespacedName).Return().Times(1)
-			err := cesReconciler.processCreateOrUpdate(selector, &testSelectorNamespacedName)
+			err := reconciler.processCreateOrUpdate(selector, &testSelectorNamespacedName)
 			Expect(err).Should(HaveOccurred())
 		})
-
 		It("CES Delete failure", func() {
-			err := cesReconciler.processDelete(&testSelectorNamespacedName)
+			err := reconciler.processDelete(&testSelectorNamespacedName)
 			Expect(err).Should(HaveOccurred())
 		})
+
+		It("CloudEntitySelector start with no CRs", func() {
+			reconciler := &CloudEntitySelectorReconciler{
+				Log:    logf.Log,
+				Client: fakeClient,
+				Scheme: scheme,
+			}
+			ctrlsync.GetControllerSyncStatusInstance().Configure()
+			ctrlsync.GetControllerSyncStatusInstance().SetControllerSyncStatus(ctrlsync.ControllerTypeCPA)
+			// Start CES reconciler with 0 CRs, and verify sync status, pendingSyncCount and initialization status.
+			err := reconciler.Start(context.TODO())
+			Expect(err).Should(BeNil())
+			Expect(reconciler.pendingSyncCount).Should(Equal(0))
+			Expect(reconciler.initialized).Should(BeTrue())
+			val := ctrlsync.GetControllerSyncStatusInstance().IsControllerSynced(ctrlsync.ControllerTypeCES)
+			Expect(val).Should(BeTrue())
+		})
```

Comment:
you wanna give new line before every IT?

Reply:
For unit-tests, don't want to add new line for each test as it can grow in a single test file.